recipes-kernel/kernel-tests/kernel-performance-tests-files/parallel-container/Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM ubuntu:22.04
+
+# Install rt-tests, sudo, etc
+RUN apt update
+RUN DEBIAN_FRONTEND=noninteractive apt install -y \
```

Comment:
Docker caches the container at each step. So index updates and install operations are normally done as a single command. eg.
```dockerfile
RUN apt update && apt install -y ...
```